src/question-renderer/question-renderer.ts

```diff
@@ -0,0 +1,62 @@
+import { QuestionType } from "../types";
```

Comment:
Theres eventually going to be more QuestionRenderer implementations so these classes should probably be moved to their own files for clarity.